exercises/concept/lasagna/.docs/introduction.md

```diff
@@ -3,122 +3,98 @@
+```
 
-julia> 16 - 6
-10
+## Functions
 
-julia> 16 * 6
-96
+There are two common ways to define a named function in Julia:
 
-julia> 16 % 6
-4
-```
+1. Using the `function` keyword
 
-#### Division
+    ```julia
+    function muladd(x, y, z)
+        x * y + z
+    end
+    ```
 
-Dividing two numbers with the `/` operator will result in a floating point value.
-To perform integer division
+    Indentation by 4 spaces is conventional for readability, but the compiler ignores this.
+    The `end` keyword is essential.
 
-- use the `div(x, y)` function, or
-- use the `÷` operator (Julia source code is unicode-aware)
+    Note that we could have written `return x * y + z`.
+    However, Julia functions always return the last expression evaluated, so the `return` keyword is optional.
+    Many programmers prefer to include it to make their intentions more explicit.
 
-```julia-repl
-julia> 16 / 6
-2.6666666666666665
-
-julia> div(16, 6)
-2
+2. Using the "assignment form"
 
-julia> 16 ÷ 6
-2
-```
+    ```julia
+    muladd(x, y, z) = x * y + z
+    ```
 
-~~~~exercism/note
-It's natural to use Unicode symbols in Julia source files, typically in mathematical expressions.
-When using the Julia REPL, or in other Julia editing environments, the division symbol can be entered by typing `\div` followed by the `Tab` key.
-More details can be found in the manual at [Unicode Input][unicode].
+    This is most commonly used for making concise single-expression functions.
 
-[unicode]: https://docs.julialang.org/en/v1/manual/unicode-input/#Unicode-Input
-~~~~
+    A `return` keyword is *never* used in the assignment form.
 
-## Comments
+The two forms are equivalent, and are used in exactly the same way, so choose whichever is more readable.
 
-Julia supports two kinds of comments.
-Single line comments are preceded by `#` and multiline comments are inserted between `#=` and `=#`.
+Invoking a function is done by specifying its name and passing arguments for each of the function's parameters:
 
 ```julia
-add(1, 3) # returns 4
+# invoking a function
+muladd(10, 5, 1)
 
-#= Some random code that's no longer needed but not deleted
-sub(x, y) = x - y
-mulsub(x, y, z) = sub(mul(x, y), z)
-=#
+# and of course you can invoke a function within the body of another function:
+square_plus_one(x) = muladd(x, x, 1)
 ```
 
-## Types
+## Naming conventions
+
+Like many languages, Julia requires that names (of variables, functions, and many other things) start with a letter, followed by any combination of letters, digits and underscores.
 
-Depending on which other programming languages you know, you may expect parameters, variables or return values to have explicit type annotations.
-For now, assume that Julia will infer the types automagically and don't worry about them, we will get to the specifics of the type system in later exercises.
+By convention, variable, constant, and function names are *lowercase*, with underscores kept to a reasonable minimum.
```

Comment:
Maybe some examples just to be clear?

Reply:
This feels like something that should be expanded in `basics/about.md`. I'm trying hard to keep `introduction.md` to the bare minimum needed to solve the exercise. Jeremy has said repeatedly that this is what he wants, and I'm not disagreeing.